domenetjenester/beregningsgrunnlag/src/main/java/no/nav/foreldrepenger/domene/migrering/BeregningMigreringTjeneste.java

```diff
@@ -127,10 +127,21 @@ private void sammenlignGrunnlag(MigrerBeregningsgrunnlagResponse response, Behan
         var fpJson = StandardJsonConfig.toJson(fpsakGrunnlag);
         var kalkJson = StandardJsonConfig.toJson(kalkulusGrunnlag);
         if (!fpJson.equals(kalkJson)) {
+            logg(fpJson, kalkJson);
             throw new IllegalStateException("Missmatch mellom kalkulus json og fpsak json av samme beregninsgrunnlag");
         }
     }
 
+    private void logg(String jsonFpsak, String jsonKalkulus) {
+        try {
+            var maskertFpsak = jsonFpsak.replaceAll("\\d{13}|\\d{11}|\\d{9}", "*");
+            var maskertKalkulus = jsonKalkulus.replaceAll("\\d{13}|\\d{11}|\\d{9}", "*");
+            LOG.info("Json fpsak: {} Json kalkulus: {} ", maskertFpsak, maskertKalkulus);
```

Comment:
Vi kunne kanskje logget til securelogg om det skulle blir mismatch på fnr nivå

Reply:
Ja, det hadde vært mulig. Det skal ikke være noe fnr i grunnlaget, men det kan inneholde både aktørId og orgnr. Kan starte med å sladde det og se om vi har behov for å se de, så kan vi logge til securelog?

Reply:
Sounds like a good plan :)